src/sql/base/browser/ui/taskbar/actionbar.ts

```diff
@@ -128,15 +143,87 @@ export class ActionBar extends ActionRunner implements IActionRunner {
 		this._actionsList = document.createElement('ul');
 		this._actionsList.className = 'actions-container';
 		this._actionsList.setAttribute('role', 'toolbar');
+		this._actionsList.id = 'actions-container';
 		if (this._options.ariaLabel) {
 			this._actionsList.setAttribute('aria-label', this._options.ariaLabel);
 		}
 
 		this._domNode.appendChild(this._actionsList);
 
+		if (this._collapseOverflow) {
+			this._overflow = document.createElement('ul');
+			this._overflow.id = 'overflow';
+			this._overflow.className = 'overflow';
+			this._domNode.appendChild(this._overflow);
+		} else {
+			this._actionsList.style.flexWrap = 'wrap';
+		}
+
 		container.appendChild(this._domNode);
 	}
 
+	@debounce(300)
+	private resizeToolbar() {
+		let width = document.getElementById('actions-container').offsetWidth;
+		let fullWidth = document.getElementById('actions-container').scrollWidth;
+
+		// collapse actions that are beyond the width of the toolbar
+		if (width < fullWidth) {
+			// create '•••' more element if it doesn't exist yet
+			if (!this._moreItemElement) {
+				this._moreItemElement = document.createElement('li');
+				this._moreItemElement.className = 'action-item more';
+				this._moreItemElement.setAttribute('role', 'presentation');
+				this._moreItemElement.id = 'more';
+				const innerText: HTMLElement = document.createElement('a');
+				innerText.setAttribute('role', 'button');
+				innerText.innerHTML = '...';
```

Comment:
I was referring to the ... on top of the connection view. it is using codicon
<img width="264" alt="Screen Shot 2020-04-01 at 11 42 39 AM" src="https://user-images.githubusercontent.com/13777222/78174357-f8747500-740d-11ea-8a7b-45b16e8763ce.png">